src/styles/theme.js

```diff
@@ -49,6 +49,7 @@ export const colors = {
   silverTree: 'rgba(112, 189, 158, 0.40)',
   finn: 'rgba(90, 47, 86, 0.3)',
   fountainBlue: '#48B2B7',
+  fountainBlue2: '#6CC1C5',
```

Comment:
I don't like this name haha

Reply:
I mean I want it to be descriptive, like is it lighter or what is the difference? is it almost the same? In case it is, should it be the same, is it a design mistake, have you checked?

Reply:
it is lighter, I updated :)